config/auth_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) OpenFaaS Author(s) 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package config
+
+// import (
```

Comment:
Hope to see these fixed up in a following commit.

Reply:
This has been removed.